packages/terra-form-select/src/_NoResults.jsx

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
+import { injectIntl, intlShape } from 'react-intl';
 import styles from './_NoResults.module.scss';
 
 const cx = classNames.bind(styles);
 
 const propTypes = {
+  /**
+   * The intl object containing translations. This is retrieved from the context automatically by injectIntl.
```

Comment:
We should add the `@private` syntax so this doesn't show up in the generated prop-types documentation.